packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -19,7 +19,7 @@ use crate::{
         crud::AllLatest,
         error::{EntityDoesNotExist, QueryError},
     },
-    GraphPool,
+    GraphPool, PersistedEntity,
 };
 
 #[derive(Component, Serialize)]
```

Comment:
We still have `QualifiedEntity` around and return that for create and update. Maybe it's not worth it to change these to also return persisted just yet as it would be a little more work we don't need done yet. But just pointing it out

Reply:
As discussed offline, have gotten rid of `QualifiedEntity` and replaced with an identifier instead